tags/state/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@marko-tags/state",
+  "description": "State manipulation in your template",
```

Comment:
The description should probably mention Marko. Eg `Inline state management within your Marko templates.`